.travis.yml

```diff
@@ -8,4 +8,13 @@ script:
   - npm run lint
   - npm run test
   - codecov
-cache: yarn
\ No newline at end of file
+cache: yarn
+
+deploy:
+  provider: npm
+  on:
+    tags: true
+    branch: master
+  email: 'lezhi.li@uber.com'
+  api_key:
```

Comment:
Although I've never done deploy in this way, I'm not sure this is a good idea to include your personal api key at here. Also, I feel it's a good idea to deploy to npm automatically when bumping the version.

Reply:
Deploying using Travis CI is still quite a new thing to me, not so sure what's the benefit of it versus deploying using lerna or ocular.

Reply:
@javidhsueh these seem to be public keys https://docs.travis-ci.com/user/encryption-keys/.

Reply:
@javidhsueh that is not my personal API key, that's an [encrypted](https://docs.travis-ci.com/user/deployment/npm/#npm-auth-token) API key by travis; and the behavior of `tag: true` is exactly deploying only [after version bump](https://docs.travis-ci.com/user/deployment/npm/#what-to-release).

Reply:
@kenns29 the benefit is that it automatically deploys if it detects a version bump, without human interference. I believe both lerna and ocular require either manually typing the command or additional hook to CI tools.